ztools/http/client.go

```diff
@@ -613,6 +613,10 @@ func (c *Client) Do(req *Request) (resp *Response, err error) {
 					timeout: true,
 				}
 			}
+			if err == io.EOF {
```

Comment:
What error type is actually returned? Can you cast it to it's real type and then check if it contains an io.EOF, or is that not possible?